From f3f92d60b1299f43e9d0f36f7f311f85d9e13b2f Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 11:11:49 +0100
Subject: [PATCH 33/53] test_1_to_1_recv: minor: fix compiler warning

We don't need both, since we test afterwards anyway; if it would fail, also the
test would not pass.

$ ./test_1_to_1_recvfrom
test_1_to_1_recvfrom.c  1 PASS : recvfrom() with a bad socket descriptor - EBADF
test_1_to_1_recvfrom.c  2 PASS : recvfrom() with invalid socket - ENOTSOCK
test_1_to_1_recvfrom.c  3 PASS : recvfrom() with invalid message ptr - EFAULT
test_1_to_1_recvfrom.c  4 PASS : recvfrom() on listening socket - ENOTCONN
test_1_to_1_recvfrom.c  5 PASS : recvfrom() on a socket that has received shutdown - EOF
test_1_to_1_recvfrom.c  6 PASS : recvfrom() on a socket with pending message that has sent shutdown - SUCCESS
test_1_to_1_recvfrom.c  7 PASS : recvfrom() on a socket with no pending messages and  has sent shutdown - ENOTCONN

$ ./test_1_to_1_recvmsg
test_1_to_1_recvmsg.c  1 PASS : recvmsg() with a bad socket descriptor - EBADF
test_1_to_1_recvmsg.c  2 PASS : recvmsg() with invalid socket - ENOTSOCK
test_1_to_1_recvmsg.c  3 PASS : recvmsg() with invalid iovec ptr - EFAULT
test_1_to_1_recvmsg.c  4 PASS : recvmsg() with invalid msghdr ptr - EFAULT
test_1_to_1_recvmsg.c  5 PASS : recvmsg() on listening socket - ENOTCONN
test_1_to_1_recvmsg.c  6 PASS : recvmsg() on a socket that has received shutdown - EOF
test_1_to_1_recvmsg.c  7 PASS : recvmsg() on a socket with pending message that has sent shutdown - SUCCESS
test_1_to_1_recvmsg.c  8 PASS : recvmsg() on a socket with no pending messages and  has sent shutdown - ENOTCONN

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/func_tests/test_1_to_1_recvfrom.c | 4 ++--
 src/func_tests/test_1_to_1_recvmsg.c  | 3 +--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/func_tests/test_1_to_1_recvfrom.c b/src/func_tests/test_1_to_1_recvfrom.c
index a45c228..a4bdf6a 100644
--- a/src/func_tests/test_1_to_1_recvfrom.c
+++ b/src/func_tests/test_1_to_1_recvfrom.c
@@ -63,7 +63,7 @@ int TST_CNT = 0;
 int
 main(int argc, char *argv[])
 {
-        int ret, msg_count;
+        int msg_count;
 	socklen_t len;
 	int sk,pf_class,lstn_sk,acpt_sk, flag;
         char *message = "hello, world!\n";
@@ -150,7 +150,7 @@ main(int argc, char *argv[])
 
 	count = test_send(acpt_sk, message, msg_count, flag);
 
-	ret = test_shutdown(sk, SHUT_WR);
+	test_shutdown(sk, SHUT_WR);
 
 	/*recvfrom () TEST5:reading on a socket that received SHUTDOWN*/
 	count = recvfrom(acpt_sk, message_rcv, msg_count, flag,
diff --git a/src/func_tests/test_1_to_1_recvmsg.c b/src/func_tests/test_1_to_1_recvmsg.c
index f017728..ceccc31 100644
--- a/src/func_tests/test_1_to_1_recvmsg.c
+++ b/src/func_tests/test_1_to_1_recvmsg.c
@@ -65,7 +65,6 @@ int
 main(int argc, char *argv[])
 {
         socklen_t len;
-	int ret;
 	int sk,pf_class,lstn_sk,acpt_sk;
 	int flag = 0;
 	struct msghdr inmessage;
@@ -165,7 +164,7 @@ main(int argc, char *argv[])
 
 	count = test_send(acpt_sk, message1, strlen(message), 0);
 
-	ret = test_shutdown(sk, SHUT_WR);
+	test_shutdown(sk, SHUT_WR);
 
 	flag = MSG_NOSIGNAL;
 	/*recvmsg () TEST6:reading on a socket that received SHUTDOWN*/
-- 
1.7.11.7

